docs/DeveloperGuide.md

```diff
@@ -251,6 +251,33 @@ _{more aspects and alternatives to be added}_
 _{Explain here how the data archiving feature will be implemented}_
 
 
+### Help feature
+#### Steps to trigger
+1. User opens the app
+2. User keys in `help`
+#### Implementation
+1. When the user enters the term help. it triggers the help feature in the parser under the switch case.
+2. After it is triggered, it will display a short list of possible commands that the user can use.
```

Comment:
Apart from that, it will also open the user guide in a default browser. Can consider to include this to be more precise and to avoid confusion when the user sees the user guide instead of the short list of possible commands. 😸

Reply:
Ok changed the help, the diagrams maybe will add in the next iteration.